examples/greet/welcome_command.cr

```diff
@@ -4,6 +4,8 @@ class WelcomeCommand < Cling::Command
     @summary = @description = "sends a friendly welcome message"
 
     add_argument "name", description: "the name of the person to greet", required: true
+    add_option 'h', "help", description: "sends help information"
```

Comment:
Also added the help option to the example, as it was referenced, but not actually included, leading to an error if the user ran it.